va/va_dec_vp9.h

```diff
@@ -189,8 +189,22 @@ typedef struct  _VADecPictureParameterBufferVP9
      */
     uint8_t                 bit_depth;
 
+    /** \brief loop filter scemantics
+     */
+    uint8_t                 mode_ref_delta_enabled;
+    uint8_t                 mode_ref_delta_update;
```

Comment:
(I'm guessing these two fields are intended to be the spec variables `loop_filter_delta_enabled` and `loop_filter_delta_update`, since these are the names of the matching variables in AV1.)
Why?  The loop filter levels are present per-segment in `VASegmentParameterVP9.filter_level`, in which the adjustment fields these are indicating the presence of have already been applied.